### ISSUE TITLE
[BUG] Names missing from group chat

### ISSUE THREAD

**Issue by kylerankin** [nanu-c/axolotl#861]
### Description

Starting yesterday, group chat no longer listed the names of each chat participant along with their message. This occurred within an existing group chat in the running client, without a restart. I am running the latest 1.2.0-1 arm64 debian asset from the official release builds.

I did attempt to restart the client but without any change. Even now I can scroll back and see the point where the chat stopped showing names associated with messages.

### Steps to Reproduce

1. Use Group Chat
2. See whether messages in group chat contain the names of the senders

**Expected behavior:** Group chat messages should display the name of the sender

**Actual behavior:** Group chat messages only display the messages and not the name of the sender

#### Versions

Axolotl Beta 1.2.0

#### Device
Librem 5

#### Link to Debug Log
No debug log found in recommended location

**Comment by kylerankin** [comment]
I inspected the messages table in the sqlite database and at the point where messages stopped showing names, the source column in messages rows stopped being populated.

This bug has now also extended to adding new contacts and chatting with them. New contacts also don't show the name of the contact within the chat. I was able to fix this in one instance by editing the appropriate row in the sessions table by hand, as a test, but obviously this isn't a solution.

Possibly something changed in the network such that message sources aren't being parsed correctly? Messages do update the appropriate session (so 1:1 chats do appear in the right places, and group chats as well), but they no longer show the source in the messages table.
